homeassistant/components/zwave_js/__init__.py

```diff
@@ -21,6 +22,7 @@
 from zwave_js_server.model.value import Value, ValueNotification
 
 from homeassistant.components.hassio import AddonError, AddonManager, AddonState
+from homeassistant.components.logger import EVENT_LOGGING_CHANGED
```

Comment:
This constant should probably move so we don't need to depend on the logger integration. Events go through the core and should not require a dependency.

Reply:
https://github.com/home-assistant/core/pull/100974

let me know which order you'd like to merge these in. We can merge this and then rebase the other and fix, or vice versa